meteor/client/lib/ReactMeteorData/ReactMeteorData.tsx

```diff
@@ -370,6 +372,46 @@ export function useTracker<T, K extends undefined | T = undefined>(
 	return meteorData
 }
 
+function useReadyState(): {
+	ready: boolean
+	setReady: (value: boolean) => void
+	cancelPreviousReady: (timeout: number) => void
+} {
```

Comment:
Minor note 1: It would be good to add a description and maybe a example use case for this one?

Minor note 2: The implementation of this doesn't really have anything to do with a "ready" state.
I think I would have understood it quicker if it had been a generic "state with a delayed reset method"-thing.
Perhaps rename it to something like the following (and let the state value be anything, and the reset-value be `undefined`)?

```suggestion
function useDelayState<T>(): {
	value: T | undefined
	setState: (newValue: T) => void
	resetState: (delay: number) => void
} {
```